Sources/NIOHTTP2/HTTP2FrameParser.swift

```diff
@@ -923,6 +929,7 @@ struct HTTP2FrameDecoder {
 struct HTTP2FrameEncoder {
     private let allocator: ByteBufferAllocator
     var headerEncoder: HPACKEncoder
+    var maxFrameSize: UInt32 = 1<<14
```

Comment:
I am not against using `<<` but I think most of the times people prefer to have the "real value" here. If you want to use `<<` maybe add some link / comment why this is a good default value.

Reply:
Heh, it's a good default value because it's what the spec requires. ;) Will add.

Reply:
Thats what I assumed :D So adding a link to the spec is what we need :)